src/views/map/layers/mission-info.jsx

```diff
@@ -78,6 +82,36 @@ import {
 import MissionSlotTrajectoryFeature from '~/views/map/features/MissionSlotTrajectoryFeature';
 import UAVTrajectoryFeature from '~/views/map/features/UAVTrajectoryFeature';
 
+// -- Custom selectors
+
+const getMinimumHomePositionDistanceSelector = createSelector(
+  getMissionType,
+  (type) =>
+    type === MissionType.SHOW
+      ? getMinimumDistanceBetweenTakeoffPositionsInShow
+      : getMinimumDistanceBetweenHomePositionsInMission
+);
+
+const getMinimumDistanceBetweenHomePositions = createSelector(
+  (state) => state,
```

Comment:
This selector would become invalid when _any_ property changes _anywhere_ in the state, even if that property is completely unrelated to the minimum distance between home positions. This kinda defeats the whole point of using `createSelector(...)`.

Give me a day or so to decide whether I can come up with a better solution or not.